xmtp_mls/src/groups/summary.rs

```diff
@@ -184,6 +195,20 @@ impl ProcessSummary {
         !self.errored.is_empty()
     }
 
+    /// the last message procesed
+    pub fn last(&self) -> Option<u64> {
+        let mut v = self.total_messages.iter().copied().collect::<Vec<_>>();
+        v.sort();
```

Comment:
Rather than sorting, which can be kind of expensive, why don't we just find the highest?

Reply:
true, i'll add that to the next PR